fix #4, #5 Describe introduction, installation, quick start and so on (#9)
oam-dev/cloud-provider: alibabacloud/ros/README.md

# ROS OAM Framework

## Introduction

`ROS OAM Framework` is an implementation of Alibaba Cloud resource orchestration
that follows OAM standards. It is based on [ROS](https://www.alibabacloud.com/help/doc-detail/28852.html),

> and you can easily orchestrate various service resources of Alibaba Cloud through OAM templates.

Reviewer: ```suggestion
and you can easily orchestrate various service resources of Alibaba Cloud through OAM spec.
```

## Installation & Running

### By using [Helm v3](https://github.com/helm/helm/releases)

```shell script
helm install ros ./charts/ros --set accessKey=<AccessKeyId>,secretKey=<AccessKeySecret>
```

### Or by running go run

```shell script
go run cmd/ros/main.go --access-key-id=<AccessKeyId> --access-key-secret=<AccessKeySecret>
```

## Quick Start
> Please make sure you have a kubernetes cluster running.

After installation and running, let's start with an example which creates 
Alibaba Cloud [SLS](https://www.alibabacloud.com/help/doc-detail/48869.htm) project, logstore and index.

> ### Write Templates

Reviewer: ```suggestion
### Write OAM Configurations
```

In `example/sls`, there are server yaml files which follow OAM standards:
- `appconf_sls.yaml` is an OAM application which specify three SLS components
- `comp_sls_project.yaml` is an OAM component which indicates Alibaba Cloud SLS project
- `comp_sls_logstore.yaml` is an OAM component which indicates Alibaba Cloud SLS logstore
- `comp_sls_index.yaml` is an OAM component which indicates Alibaba Cloud SLS index

> You can refer to these files to write your own OAM template filesto create 

Reviewer: ```suggestion
These files will convert to ROS template by this controller and create
```

the Alibaba Cloud resources you want.

> ### Create Resources by Templates

Reviewer: Change templates to OAM Configurations.

all same in below instructions

    }
```

> - Generate workloads which will be generated to current 'workloads' path.

Reviewer: ```suggestion
- Sync workloads will fetch all resource info and generate workloads to current 'workloads' path.
```

```shell script

> go run gen.go -i <AccessKeyId> -s <AccessKeySecret>

Reviewer: add:

You can apply them again to update this info in cluster.

```
kubectl apply -f workloads/
```
